vms/client/nx_vms_client_desktop/src/utils/media/audio_player.cpp

```diff
@@ -306,7 +306,7 @@ bool AudioPlayer::openNonSafe( QIODevice* dataSource )
     m_mediaFileReader = mediaFileReader.release();
     m_mediaFileReader->setAudioChannel(0);
 
-    m_renderer = new QnAudioStreamDisplay( AUDIO_BUF_SIZE, AUDIO_BUF_SIZE / 2 );
+    m_renderer = new QnAudioStreamDisplay(AUDIO_BUF_SIZE, AUDIO_BUF_SIZE / 2, /*withAnalyzer*/ false, /*decodeOnly*/ false);
```

Comment:
Line is longer than 100 symbols
```suggestion
    m_renderer = new QnAudioStreamDisplay(
        AUDIO_BUF_SIZE,
        AUDIO_BUF_SIZE / 2,
        /*withAnalyzer*/ false,
        /*decodeOnly*/ false);
```